socorro/lib/migrations.py

```diff
@@ -11,17 +11,36 @@
 
 import os
 
-
-def load_stored_proc(op, filelist):
+def get_local_filepath(filename):
     """
-    Takes a list of files and the alembic op object as arguments
-    Load and run CREATE OR REPLACE function commands from files
+    Helper for finding our raw SQL files locally.
+
     Expects files to be in:
         $CWD/socorro/external/postgresql/raw_sql/procs/
     """
     app_path=os.getcwd()
+    return (app_path + '/socorro/external/postgresql/raw_sql/procs/' + filename)
```

Comment:
Depending on `os.getcwd()` makes me nervous. It means that it totally depends on where you are when you kick off the migration. And that shouldn't be necessary because we can figure out what the path is in relation to this file. E.g.
```python
procs_dir = os.path.normpath(os.path.join(
    __file__,
    '../../', 
    'external/postgresql/raw_sql/procs'
))
```